docs/docs.md

```diff
@@ -39,13 +35,9 @@ Create a Dynamics client from environment variables:
 2. DYNAMICS_TOKEN_URL
 3. DYNAMICS_CLIENT_ID
 4. DYNAMICS_CLIENT_SECRET
-
-
-5. *DYNAMICS_SCOPE (Optional)*
+5. DYNAMICS_SCOPE
```

Comment:
See [this comment](https://github.com/MrThearMan/dynamics-client/pull/5/files#r1160467202).